Update inkstroke_id.md

Eversole opened the pull request:
I counted on the documentation about the Id not changing through serialization/deserialization when building my app and found that to not be true. It also appears that it restarts the counter (at 1) upon load into a new InkStrokeContainer. I believe that the ID being unique only applies to a particular InkStrokeContainer instance. It would be nice to have some clarification about what is unique about the ID...

Eversole committed:
Update inkstroke_id.md

I counted on the documentation about the Id not changing through serialization/deserialization and found that to not be true. It also appears that it restarts the counter (at 1) upon load. I believe that the ID being unique only applies to a particular InkContainer instance. It would be nice to have some clarification about what is unique about the ID...

mattwojo commented:
@Karl-Bridge-Microsoft

Karl-Bridge-Microsoft commented:
Hey Matt, I’ve asked the FT to have a look at this. It contradicts the existing comment, so I want to confirm that it’s correct and hopefully get the brief explanation requested by the OP.

karl bridge
senior content developer
content experience r&d
wdg

From: Matt Wojciakowski [mailto:notifications@github.com]
Sent: Thursday, October 19, 2017 5:43 PM
To: MicrosoftDocs/winrt-api <winrt-api@noreply.github.com>
Cc: Karl Bridge <kbridge@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [MicrosoftDocs/winrt-api] Update inkstroke_id.md (#187)


@Karl-Bridge-Microsoft<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkarl-bridge-microsoft&data=02%7C01%7Ckbridge%40microsoft.com%7Cd40212d3868c453026d208d517538133%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636440569772824750&sdata=mSDFJXOeKomeJ6p7yOSpdw%2BWrwGWXYSsTJXqIQRqsY4%3D&reserved=0>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fwinrt-api%2Fpull%2F187%23issuecomment-338075573&data=02%7C01%7Ckbridge%40microsoft.com%7Cd40212d3868c453026d208d517538133%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636440569772824750&sdata=4bDsz9qNFP%2F%2F%2By%2FERGnXktgWATk4n2t4GDmgrCLkEH0%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAMQR58ljs_8QQ5pcxJkM-HcaqOKY57DVks5st-yPgaJpZM4P_n-g&data=02%7C01%7Ckbridge%40microsoft.com%7Cd40212d3868c453026d208d517538133%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636440569772824750&sdata=3ovd5BwhLUMMGF%2B1XP38b3ZoK600dwAuC2raNAIiKgo%3D&reserved=0>.

mikematteson commented:
@Karl-Bridge-Microsoft : PR **passed** review checklist. Please evaluate further.

Karl-Bridge-Microsoft committed:
Update inkstroke_id.md

Thanks for your contribution to the Windows docs! I've accepted your PR with a few revisions.
